morpho5/datastructures/object.c

```diff
@@ -219,6 +220,16 @@ varray_value *object_functiongetconstanttable(objectfunction *func) {
     return NULL;
 }
 
+/** Does a function have variadic args? */
+bool object_functionhasvargs(objectfunction *func) {
+    return (func->varg>=0);
+}
+
+/** Sets the parameter number of a variadic argument */
+void object_functionsetvarg(objectfunction *func, unsigned int varg) {
+    func->varg=varg;
```

Comment:
I don't expect to ever come up but we are assigning an unsigned int to an int which could overflow.